automated-testing/unit-testing/custom-connector.md

```diff
@@ -0,0 +1,11 @@
+# Custom Connector Testing
+
+When developing Custom Connectors to put data into the Power Platform there are some strategies you can follow:
```

Comment:
Can you elaborate more on the challenges one might face when writing tests for custom connectors?